products.css

```diff
@@ -0,0 +1,65 @@
+* {
+  font-family: "Inter", sans-serif;
+}
+
+.products-layout {
+  margin-top: 67px;
+  display: grid;
+  grid-template-columns: 250px 1fr;
```

Comment:
As Vikram told in his sessions use rems if the sizing is more than 10px

Reply:
Yes, changed it!